makefile.shade

```diff
@@ -1,8 +1,73 @@
+use namespace="System.IO"
+use namespace="System.IO.Compression"
+use namespace="System.Linq"
 
+default CONFIGURATION='${E("Configuration")}'
+default BASE_DIR_LOCAL='${Directory.GetCurrentDirectory()}'
+default BUILD_DIR_LOCAL='${Path.Combine(BASE_DIR_LOCAL, "artifacts", "build")}'
+default TOOL_EXE_NAME='dotnet-razor-tooling.exe'
 var VERSION='0.1'
 var FULL_VERSION='0.1'
 var AUTHORS='Microsoft Open Technologies, Inc.'
 -BuildQuality = "preview2";
 
 use-standard-lifecycle
-k-standard-goals
\ No newline at end of file
+k-standard-goals
+
+#repack-x86 target='compile'
+    @{    
+        if (string.IsNullOrEmpty(CONFIGURATION))
+        {
+            CONFIGURATION = "Debug";
+        }
+        
+        // Forcing to lower to match the entry in the sources project.json.
+        var configurationX86 = CONFIGURATION.ToLower() + "_x86";
+        
+        foreach (var nupkg in Files.Include(Path.Combine(BUILD_DIR_LOCAL, "*.nupkg")))
+        {
+            if (nupkg.EndsWith(".symbols.nupkg", StringComparison.OrdinalIgnoreCase))
+            {
+                continue;
+            }
+            
+            Console.WriteLine("Repacking Nupkg: " + nupkg);
+            
+            var extractToDirectory = nupkg + "-temp";
+            ZipFile.ExtractToDirectory(nupkg, extractToDirectory);
+            
+            // Assuming the package version starts with 1.
+            var projectName = nupkg.Substring(0, nupkg.IndexOf(".1"));
```

Comment:
Think it would be better to specify the project names up front rather than doing this. Most other tools sit with middleware or other library packages that don't require this.